crate/content.md

```diff
@@ -1,111 +1,41 @@
-
-Crate provides several installation packages, including a supported Docker image. It fits perfectly into an orchestrated microservices environment. It acts like an ephemeral, omnipresent, persistent layer for data. Application containers access their data regardless of which host the data nodes run.
-
-[Crate](https://crate.io/)
-
 %%LOGO%%
 
-# Quick Start Example: Multihost Production Setup
-
-This is an example configuration to run in a multi-host production environment. The configuration includes the required minimum settings:
-
--	Volume mapping
--	Port mapping to localhost (run only one container per machine)
--	Unicast host discovery
-
-To start the Crate cluster in containers distributed to three hosts without multicast enabled, run this command on the first node and adapt the container and node names on the two other nodes:
-
-```console
-# HOSTS="crate1.example.com:4300,crate2.example.com:4300,crate3.example.com:4300"
-# HOST="crate1.example.com"
-# docker run -d -p 4200:4200 -p 4300:4300 \
-    --name crate1-container \
-    --volume /mnt/data:/data \
-    --ulimit nofile=65535 \
-    --ulimit memlock=9223372036854775807 \
-        crate \
-        crate \
-          -Ccluster.name=crate-cluster \
-          -Cnode.name=crate1 \
-          -Ctransport.publish_port=4300 \
-          -Cnetwork.publish_host="$HOST" \
-          -Cmulticast.enabled=false \
-          -Cdiscovery.zen.ping.unicast.hosts="$HOSTS" \
-          -Cdiscovery.zen.minimum_master_nodes=2
-```
-
-# The crate Docker Image
-
-To form a cluster from scratch, start a few instances of the Crate container as a background daemon:
-
-```console
-# docker run -d crate crate
-```
-
-To access the admin UI, map port 4200 and point your browser to port tcp/4200 of a node of your choice while you start it or look up its IP later:
-
-```console
-# firefox "http://$(docker inspect --format='{{.NetworkSettings.IPAddress}}' $(docker run -d crate crate)):4200/admin"
-```
-
-For production use it's strongly recommended to use only one container per machine. This will give the best possible performance and by mapping the ports from the Docker container to the host it acts like a native installation. Crate's default ports 4200 (HTTP) and 4300 (Transport protocol).
-
-```console
-# docker run -d -p 4200:4200 -p 4300:4300 crate crate
-```
-
-## Attach Persistent Data Directory
-
-Crate stores all important data in */data*. It's advised to mount this directory to avoid writing within the docker image:
-
-```console
-# docker run -d -v <data-dir>:/data crate crate
-```
-
-## Use Custom Crate Configuration
+# What Is CrateDB?
 
-Starting with 0.55.0, Crate does no longer support providing custom configuration files. However it is still possible to mount Crate's configuration into `/crate/config/crate.yml`.
+[CrateDB](github.com/crate/crate) is a distributed SQL database that makes it simple to store and analyze massive amounts of machine data in real-time.
```

Comment:
This URL should have `https://` in front (`[CrateDB](https://github.com/crate/crate)`).